tests/test_files/scheduling/task_synchronisation.pfdl

```diff
@@ -20,7 +20,7 @@ Struct CuttingResult
     sheet_parts: SheetPart[]
 End
 
-Struct TransportOrderStep
+Struct TransportOrderStep_Struct
```

Comment:
Maybe remove the _ in the Struct and Service names here to be consistent with the other test files